va/va_test.go

```diff
@@ -832,7 +832,7 @@ func TestLogRemoteDifferentials(t *testing.T) {
 			mockLog.Clear()
 
 			localVA.logRemoteDifferentials(
-				"example.com", 1999, "blorpus-01", tc.primaryResult, tc.remoteProbs)
```

Comment:
It's probably safe to remove `primaryResult` from each test case in `TestLogRemoteDifferential`.